mPemba/school-ai-challenge
# Cypress tests

## What was changed 
- Adding Cypress tests
- Adding data-cy tags to necessary elements
- Adding fixture for answer data
- Updating ReadMe 

## How to test
Start the development server
`yarn install`
`yarn dev`

In a new terminal window run:
`yarn e2e:run`

Confirm tests pass

<img width="524" alt="Screen Shot 2023-05-18 at 1 20 10 PM" src="https://github.com/mPemba/school-ai-challenge/assets/8922510/be47c13c-ad2f-4ae2-8fa7-5f90b15e6b6b">
